adage/analyze/models.py

```diff
@@ -218,3 +218,17 @@ def __unicode__(self):
         return "Model: %s, Node: %s, Gene: %s" % (self.node.mlmodel.title,
                                                   self.node.name,
                                                   self.gene.entrezid)
+
+
+class ExpressionValue(models.Model):
+    """
+    This class models the many-to-many relationship between Sample and Gene.
+    Each sample/gene pair has an expression value of floating type.
+    """
+    sample = models.ForeignKey(Sample, on_delete=models.PROTECT)
+    gene = models.ForeignKey(Gene, on_delete=models.PROTECT)
+    value = models.FloatField()
```

Comment:
Right, I guess if it's normalized then the units are taken out. Your description of the pre-processing is exactly what I was looking for.